src/main/java/cascading/tap/hive/HivePartitionTap.java

```diff
@@ -63,6 +83,23 @@ public HivePartitionTap( HiveTap parent, SinkMode sinkMode )
     super( parent, parent.getTableDescriptor().getPartition(), sinkMode );
     }
 
+  /**
+   * Constructs a new HivePartitionTap with the given HiveTap as the parent directory and sourcing only the given
+   * partitions.
+   * <p>
+   * The provided {@code partitionValues} are expected in the Hive format. For example, if the Hive table has two partition
+   * columns then the inner {@link List} is the values for all the partition columns of one partition and the outer list is
+   * for all the partitions that are required. e.g. {@code [[p1_value1, p2_value1], [p1_value2, p2_value2]]}.
+   *
```

Comment:
I don't fully understand this explanation. Could you extend this a bit with an example? 

Also, the references in the javadoc don't match with the actual variable names.